### PR TITLE
Validate Params Performs Contains on AdditionalInfo Instead Of Equals

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/34765

## Description
Changing integration validation for additionalinfo in parameter, changing its condition from equals to contained.
This is needed because some integrations do require to add more additionalinfo in their parameter, and currently
are enforced to stick to the current additionalinfo structure exactly instead of containing it

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
@tomneeman151293 Why the name is test failure?

**Comment by tomneeman151293** [comment]
@ShahafBenYakir Changed name

**Comment by tomneeman151293** [comment]
Test Failure:
https://app.circleci.com/pipelines/github/demisto/demisto-sdk/8677/workflows/3b76d42c-f52f-41b3-801a-ec23acca4a6e/jobs/35963

**Review by ShahafBenYakir — COMMENTED** [review]
Let's go over this together

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 * Added the *--include-untracked* flag to the **validate** command to include files which are untracked by git in the validation process.
 * Improved the `pykwalify` error outputs in the **validate** command.
 * Added the *--print-pykwalify* flag to the **validate** command to print the unchanged output from `pykwalify`.
+* Updated the **demisto-sdk validate** command to check that 'additionalinfo' field contains the expected 'additionalinfo' value for feed required parameters.
```
Comment (ShahafBenYakir) [inline CHANGELOG.md:13]:
```suggestion
* Updated the **demisto-sdk validate** command to check that the 'additionalinfo' field only contains the expected value for feed required parameters and not equal to it.
```

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**